zbus/src/connection/mod.rs

```diff
@@ -1863,4 +1882,41 @@ mod p2p_tests {
 
         (conn1, conn2)
     }
+
+    #[test]
+    #[timeout(15000)]
+    fn test_method_timeout() {
```

Comment:
* I just realized that you forgot that I told you on the Matrix channel that these tests don't need to be specific to p2p at all. Come to think of it, it's probably best to add a method to the e2e test (in `zbus/tests/e2e.rs`) that tests this, instead of a completely separate test.
* `test_` prefixes are redundant and inconsistent with naming of other tests (even in this same module here).

Reply:
Is it Ok to use `#[cfg(feature = "p2p")]` there? Without it I can't build Connections with no Authentication. I see there's a bunch of code there wrapped in `p2p`.

Reply:
Ok. I THINK I did it they way you want it, but I am not 100% sure.